### ISSUE TITLE
Display and Use on Microsoft Edge

### ISSUE THREAD

**Issue by btrnew** [spawaskar-cora/cora-docs#575]
When I try to access CoRA it displays weird and will not allow me to deviate from the main screen or input anything. I am using Microsoft Edge.

![error in display](https://user-images.githubusercontent.com/43450732/69192696-34dfae00-0ac9-11ea-939b-0975934f5cbd.PNG)

**Comment by SachinPawaskarUNO** [comment]
@btrnew Is is happening on other people's machines? Can you please check this first?
Because I can see some folks from Omaha and Hawaii have logged in today and used the CoRA app and nobody else has reported this.

**Comment by btrnew** [comment]
@SachinPawaskarUNO I verified with one other person today that it's not working on both Microsoft Edge and Internet Explorer. You can click to see the Dashboard page, but when you click Specimen or DNA it changes the URL but nothing changes on the screen.

**Comment by clegarde** [comment]
I am currently using in Edge without issues. Haven't heard any issues from other Offutt folks.

**Comment by btrnew** [comment]
@clegarde Out of curiosity what computer and OS are you using? I don't know if that would affect things, but it can't hurt to give Sachin the info. I've got a Dell with Windows 10 Enterprise v 1803, 64 bit operating system.

**Comment by SachinPawaskarUNO** [comment]
@btrnew @clegarde @fedamann Something has changed on the machines either OS or Browser. We need to collect more information on both the OS and browser versions. This has happened in the past as well (missing icons due to unsupported version of IE and Edge). See #370

**Comment by clegarde** [comment]
Computer:
Edition: Windows 10 Enterprise
Version: 1803
OS build 17134.1130

Edge:
Microsoft Edge 42.17134.1098.0
Microsoft EdgeHTML 17.17134

**Comment by btrnew** [comment]
My specs look the same as Carrie's, but let me know if there is other information you need. Right now we can't use CoRA in HI 

Computer: 
Windows 10 Enterprise
Version: 1803
OS build 17134.1130

Edge:
Microsoft Edge 42.17134.1098.0
Microsoft EdgeHTML 17.17134

**Comment by SachinPawaskarUNO** [comment]
I tried this with Windows and Edge from outside

![image](https://user-images.githubusercontent.com/19599773/69486359-6bdafa00-0e10-11ea-86db-d92844a7c05d.png)

**Comment by btrnew** [comment]
Checked again today and asked around. Everyone I've asked in HI is having this same issue. I noticed last week when I got into work that we got pushed some kind of update, but I have no idea what the update was. I have also noticed that I experience this issue with the display format and usage for several sites. For example if I go to physanth.org (the website for a conference we attend), this is what I get: 

![physanthscreenshot](https://user-images.githubusercontent.com/43450732/69572150-98555a00-0f67-11ea-874c-91b3c3d5b1f2.PNG)

It's as if my browsers are set to remove any advanced formatting.
